vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -28,8 +28,6 @@ using namespace xaml::Navigation;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-ReactApplication::ReactApplication() = default;
```

Comment:
>ReactApplication() = default; [](http://example.com/codeflow?start=18&length=29)

Why is it removed? After this change the constructor will be generated as a part of header and copied into each user of the header file.
When we implement constructor once in the .cpp we make the code generation more efficient because it reduces size of OBJ files, and the work needed to be done by linker.